kuma/core/templatetags/jinja_helpers.py

```diff
@@ -194,3 +195,24 @@ def get_locale_localized(locale):
     language = settings.LOCALES[locale].english
 
     return _(language)
+
+
+@library.global_function
+@lru_cache()
```

Comment:
This is a problem, since once the value for a particular flag name enters the cache, it's never cleared until it's pushed out via the LRU mechanism, and then that only happens per Django instance. So in other words, if the Waffle flag is changed via the admin console, the change most likely won't be picked-up by the Django instances, and possibly not ever until this particular LRU cache is manually cleared in each and every Django instance.